bindings/python/tools/python/ostk/astrodynamics/converters.py

```diff
@@ -1,5 +1,7 @@
 # Apache License 2.0
 
+from typing import Union
```

Comment:
Why is this necessary? Can't we use |

Reply:
I was just using `|` but the linter didn't like it.
Seems ok now though, so adapting 👍